server/src/test/java/org/opensearch/cluster/coordination/FollowersCheckerTests.java

```diff
@@ -787,4 +795,47 @@ public TransportResponse.Empty read(StreamInput in) {
         }
 
     }
+
+    public void testFollowerCheckTimeoutValue() {
+        Setting<TimeValue> setting1 = FOLLOWER_CHECK_TIMEOUT_SETTING;
+        Settings timeSettings1 = Settings.builder().put(setting1.getKey(), "60s").build();
+
+        try {
+            ClusterUpdateSettingsResponse response = client().admin()
+                .cluster()
+                .prepareUpdateSettings()
+                .setPersistentSettings(timeSettings1)
+                .execute()
+                .actionGet();
+
+            assertAcked(response);
+            assertEquals(timeValueSeconds(60), setting1.get(response.getPersistentSettings()));
+        } finally {
+            // cleanup
+            timeSettings1 = Settings.builder().putNull(setting1.getKey()).build();
+            client().admin().cluster().prepareUpdateSettings().setPersistentSettings(timeSettings1).execute().actionGet();
+        }
+    }
+
+    public void testFollowerCheckTimeoutMaxValue() {
+        Setting<TimeValue> setting1 = FOLLOWER_CHECK_TIMEOUT_SETTING;
+        Settings timeSettings1 = Settings.builder().put(setting1.getKey(), "61s").build();
+
+        try {
+            client().admin().cluster().prepareUpdateSettings().setPersistentSettings(timeSettings1).execute().actionGet();
+        } catch (IllegalArgumentException ex) {
+            assertEquals(ex.getMessage(), "failed to parse value [61s] for setting [" + setting1.getKey() + "], must be <= [60000ms]");
+        }
+    }
+
+    public void testFollowerCheckTimeoutMinValue() {
+        Setting<TimeValue> setting1 = FOLLOWER_CHECK_TIMEOUT_SETTING;
+        Settings timeSettings1 = Settings.builder().put(setting1.getKey(), "0s").build();
+
+        try {
+            client().admin().cluster().prepareUpdateSettings().setPersistentSettings(timeSettings1).execute().actionGet();
+        } catch (IllegalArgumentException ex) {
+            assertEquals(ex.getMessage(), "failed to parse value [0s] for setting [" + setting1.getKey() + "], must be >= [1ms]");
+        }
```

Comment:
nit : we could simplify this to `assertThrows`

Reply:
Done!